gradlew.bat

```diff
@@ -11,8 +11,8 @@ if "%OS%"=="Windows_NT" setlocal
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
 set DEFAULT_JVM_OPTS=
 
-set DIRNAME=%~dp0

```

Comment:
If this is manually modified could you use the gradle wrapper commend to create it?